code/modules/xenomorph/xeno_structures.dm

```diff
@@ -0,0 +1,110 @@
+/obj/structure/resin
+	layer = RESIN_STRUCTURE_LAYER
+	resistance_flags = UNACIDABLE
+
+
+/obj/structure/resin/ex_act(severity)
+	switch(severity)
+		if(1)
+			take_damage(210)
+		if(2)
+			take_damage(140)
+		if(3)
+			take_damage(70)
+
+/obj/structure/resin/attack_animal(mob/living/user)
+	return
+
+/obj/structure/resin/attack_paw(mob/user)

```

Comment:
These by default do nothing so it's safe to remove them. Only attack-hand is worth overriding to just return.